packages/services/src/terminal/default.ts

```diff
@@ -173,7 +174,7 @@ class DefaultTerminalSession implements TerminalSession.ISession {
   /**
    * Connect to the websocket.
    */
-  private _initializeSocket(): Promise<void> {
+  private _initializeSocket = (): Promise<void> => {
```

Comment:
Did this signature need to change?

Reply:
because it need to bind `this`, and settimeout will change `this` to global context